src/model/client.rs

```diff
@@ -178,8 +179,8 @@ impl Client {
             true,
             true,
             true,
-            config::TG_API_ID,
-            config::TG_API_HASH.into(),
+            APPLICATION_OPTS.get().unwrap().client_id,
+            APPLICATION_OPTS.get().unwrap().client_secret.to_owned(),
```

Comment:
Save the `ApplicationOptions` into a variable, so you don't need to unwrap twice